virttest/libvirt_xml/vm_xml.py

```diff
@@ -2041,6 +2041,17 @@ def get_feature_list(self):
             feature_list.append(feature_node)
         return feature_list
 
+    def get_dict_type_feature(self):
```

Comment:
change get_dict_type_feature to get_dict_type_cpu_feature ? since it is cpu feature specific

Reply:
Hi chunfu ,Thanks for your comment. But other related functions not mention "cpu" , Such as, `get_feature_list(), get_feature_index(), get_feature_name().` The are all in the class of `"class VMCPUXML(base.LibvirtXMLBase):`
"